distro/superbuild/cmake/externals.cmake

```diff
@@ -238,9 +265,18 @@ ExternalProject_Add(PythonQt
 
 ###############################################################################
 # ctkPythonConsole
+set(ctkPythonConsole_REPO)
+set(ctkPythonConsole_TAG)
+if(${DD_QT_VERSION} VERSION_GREATER "4")
+  set(ctkPythonConsole_REPO https://github.com/mwoehlke-kitware/ctkPythonConsole)
+  set(ctkPythonConsole_TAG qt5)
```

Comment:
@mwoehlke-kitware Mind creating a PR for branch [qt5](https://github.com/mwoehlke-kitware/ctkPythonConsole/commits/qt5) on ctkPythonConsole and mention this PR (#508) ?